stdlib/xml/dom/minidom.pyi

```diff
@@ -22,6 +22,7 @@ class Node(xml.dom.Node):
     def lastChild(self) -> Node | None: ...
     @property
     def localName(self) -> str | None: ...
+    def __bool__(self) -> bool: ...
```

Comment:
Should this perhaps return `Literal[True]`, or is the expectation that subclasses can return `False`? The method seems quite pointless to me, as objects are truthy by default, and the base class `xml.dom.Node` doesn't define `__bool__`.

Reply:
I assumed it was a subclass thing, but you're right that I don't see any subclasses in CPython.